homeassistant/components/tado/services.py

```diff
@@ -0,0 +1,52 @@
+"""Services for the Tado integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.core import HomeAssistant, ServiceCall, callback
+from homeassistant.exceptions import ServiceValidationError
+from homeassistant.helpers import selector
+
+from .const import (
+    ATTR_CONFIG_ENTRY,
+    ATTR_MESSAGE,
+    ATTR_READING,
+    DATA,
+    DOMAIN,
+    SERVICE_ADD_METER_READING,
+)
+
+_LOGGER = logging.getLogger(__name__)
+SCHEMA_ADD_METER_READING = vol.Schema(
+    {
+        vol.Required(ATTR_CONFIG_ENTRY): selector.ConfigEntrySelector(
```

Comment:
```suggestion
        vol.Required(CONF_CONFIG_ENTRY): selector.ConfigEntrySelector(
```

Reply:
why?

Reply:
We typically use `CONF_*` for configured values and `ATTR_*` for displaying attributes

Reply:
Understood. `homeassistant.const` doesn't have `CONF_CONFIG_ENTRY`. It feels a bit silly to create a duplcate constant just to satify this tipical behaviour. Do you still recommend it?

Reply:
You would rename `ATTR_CONFIG_ENTRY` and not duplicate?

Reply:
silly me :-D

Reply:
`ATTR_READING` should also be `CONF_READING`

Reply:
renamed